content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,30 @@ environment, then Prometheus is a good choice.
 
 ### Architecture
 
-Sensu has a [number of components](https://docs.sensu.io/sensu-core/latest/overview/architecture/). It uses
-RabbitMQ as a transport, Redis for current state, and a separate server for
-processing and API access.
+Sensu is a primarily agent-based monitoring solution (agentless applications are also supported) ([reference](https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/deployment-architecture/)). The Sensu agents automatically connect to Sensu backends over an encrypted pub/sub transport (TLS-encrypted websockets), and handle automated registration/de-registration, and local service discovery. Sensu's use of a pub/sub transport allows it to securely work over complex network topologies including NATs, VPNs, and "air-gapped" environments (useful in IoT deployments).
+
+Sensu agents collect observability data via [StatsD socket](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-process/aggregate-metrics-statsd/) (StatsD formatted metrics), [TCP/UDP socket](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/agent/) (Sensu events), [HTTP JSON API](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/agent/#create-observability-events-using-the-agent-api) (Sensu events), Telegraf metrics (InfluxDB line protocol), Prometheus Exporters (Prometheus Exposition Format), or by [executing local check scripts or other plugins](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/agent/#create-observability-events-using-service-checks) and capturing the resulting standard output and exit status code. 
 
-All components of a Sensu deployment (RabbitMQ, Redis, and Sensu Server/API) can be clustered for highly available and redundant configurations.
+The entire Sensu observabliity pipeline configuration can be defined as code (declarative JSON or YAML template files). All components of a Sensu deployment can be clustered for highly available and scalable configurations beyond 50,000 servers (or entities).
 
 ### Summary
-If you have an existing Nagios setup that you wish to scale as-is, or want to take advantage of the automatic registration feature of Sensu, then Sensu is a good choice.
 
-If you want to do whitebox monitoring, or have a very dynamic or cloud based environment, then Prometheus is a good choice.
+If you have an existing Nagios (or similar) monitoring solution in a heterogeneus ephemeral cloud environment (e.g. EC2, GCP, Azure, Kubernetes, etc), or if you wish to consolidate and integrate existing best-of-breed monitoring solutions, then you should consider Sensu with its automatic registration/de-registration and data transformation features.
```

Comment:
Is this not suitable for Nagios in homogeneous, persistent, or on-prem environments? I think the previous wording covers this part better, as you seem to be over-constraining where Sensu may make sense.

The 2nd half reads as marketing, can you propose something more concrete?